File: leetcode/leetcode412.cpp
#include <iostream>
#include <vector>
using namespace std;
class Solution
{
public:
	vector<string> fizzBuzz(int n){
		std::vector<string> v;
		for (int i = 1; i <= n; i++)
		{
			/* code */
			if (i%3==0 && i%5==0)
				v.push_back("FizzBuzz");
			else if (i%3==0)
			{
				v.push_back("Fizz");
			}
				else if (i%5==0)
				{
					/* code */
					v.push_back("Buzz");
				}
					else
						v.push_back(to_string(i));
		}
		return v;
	}
};
int main()
{
	Solution a;
	vector<string> b;
	b=a.fizzBuzz(15);
	for (int i = 0; i < b.size(); ++i)
	{
		/* code */
		cout<<b[i];
	}	
	return 0;
}

File: leetcode/leetcode412.py
#!/usr/env/python
#coding=utf-8
#Fizz Buzz 题意：字符串输出1 到 n，如果这个数是3的倍数则输出‘Fizz’，如果是5
#的倍数则是Buzz，如果是3 和 5共同的倍数则输出'Fizzbuzz'
class Solution(object):
	"""docstring for """
	def fizzBuzz(self,n):
		cout=[]
		for x in xrange(1,n+1):
			if x%3==0 and x%5==0:
				cout.append('Fizzbuzz')
			elif x%3==0:
				cout.append('Fizz')
			elif x%5==0:
				cout.append('Buzz')
			else:
				cout.append(str(x))
		return cout
# a=Solution()
# print(a.fizzBuzz(15))				